scripts/visualize/visualizeFutureWarming.R

```diff
@@ -2,41 +2,97 @@
 library(RColorBrewer)
 library(plyr)
 library(xml2)
+library(grid)
+library(maps)
+library(ggplot2)
 
 visualizeData.visualizeFutureWarming <- function(processedFutureWarming, outfile, ...){
 
   all_period_gdd = processedFutureWarming
+  
+  #should theoreticaly pull such data import sections out in vizlab
+  wbic_latlon = read.table('data/WI_Lakes_WbicLatLon.tsv', sep='\t', header=TRUE, as.is=TRUE)
+  wisco = map_data('state', region = 'wisconsin')
+  
   lakeids = c('2331600', '805400','146100') #Exemplar lakes: Trout Lake, Mendota, Green Lake
   lakenames = c('Trout Lake', 'Lake Mendota', 'Green Lake')
+  cols = c("#377EB8", "#E41A1C" ,"#4DAF4A")
   periods = list(1989:2014, 2040:2064, 2065:2089)
   starts = sapply(periods, min)
-  cols = RColorBrewer::brewer.pal(3, 'Set1')
-  
+  #cols = RColorBrewer::brewer.pal(length(lakeids), 'Set1')
+  names(cols) = lakeids
 
   threelake = subset(all_period_gdd, lakeid %in% lakeids)
   
   grDevices::svg(filename = outfile, width=9, height=6)
-  par(mai = c(0.8,0.8, 0.2, 0.8))
+  par(mai = c(0.8, 0.8, 0.2, 0.8))
   plot(1, NA, ylim=c(2200,3300), xlim=c(0.75,3.25), xaxt='n', xlab='Years', ylab='Degree Days')
   
   #points(med_gdd~periodi, all_period_gdd, pch=16, col=rgb(0.5,0.5,0.5,0.2), cex=0.5)
   
+  theme_future <- function(base_size = 10, base_family = "")
```

Comment:
separate file, not clear how to best deal with lots of cross-file dependencies in Vizlab, so I left it in here. Plus, there are subtle differences between the themes.